src/assets/PugAsset.js

```diff
@@ -0,0 +1,18 @@
+const HTMLAsset = require('./HTMLAsset');
+const localRequire = require('../utils/localRequire');
+
+class TypeScriptAsset extends HTMLAsset {
```

Comment:
`TypeScriptAsset`  still here from copied file.

Reply:
My bad, couldn't test it because I have node v6